### PR TITLE
Added documentation for the electra smart integration

### PR DESCRIPTION
**Pull request by jafar-atili**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:  https://github.com/home-assistant/core/pull/70361
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/3708
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
It seems that this PR is targeted against an incorrect branch. Documentation updates which apply to our current stable release should target the `current` branch. Please change the target branch of this PR to `current` and rebase if needed. If this is documentation for a new feature, please add a link to that PR in your description.

File: source/_integrations/electrasmart.markdown [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: SwitchBee
```
Comment (thecode) [inline source/_integrations/electrasmart.markdown]:
```suggestion
title: Electra Smart
```

**Review by thecode — CHANGES_REQUESTED** [review]

File: source/_integrations/electrasmart.markdown [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+ha_domain: electrasmart
+ha_platforms:
+  - climate
+
+ha_integration_type: integration
+---
+
+[Electra Air](https://www.electra-air.co.il), is a company which manufactures and sell Air Conditioners.
+
+In order to setup this integration, you must have access to the phone number used and registered in the Electra Smart mobile app.
```
Comment (thecode) [inline source/_integrations/electrasmart.markdown]:
```suggestion
In order to setup this integration, you must have access to the phone number used to register in the Electra Smart mobile app.
```

**Review by thecode — APPROVED** [review]

File: source/_integrations/electrasmart.markdown [hunk 3]
```diff
@@ -0,0 +1,24 @@
+---
+title: Electra Smart
+description: Instructions for how to integrate Electra Air Conditioners within Home Assistant.
+ha_category:
+  - Climate
+ha_release: 2022.11
```
Comment (frenck) [inline source/_integrations/electrasmart.markdown:6]: This seems to be incorrect? This is released in the past?

CC @jafar-atili @thecode

File: source/_integrations/electrasmart.markdown [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@jafar-atili'
+ha_domain: electrasmart
+ha_platforms:
+  - climate
+
```
Comment (frenck) [inline source/_integrations/electrasmart.markdown:14]: Stray newline

File: source/_integrations/electrasmart.markdown [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+ha_integration_type: integration
+---
+
+[Electra Air](https://www.electra-air.co.il), is a company which manufactures and sell Air Conditioners.
```
Comment (frenck) [inline source/_integrations/electrasmart.markdown:18]: Typo: `sell` -> `sells`

File: source/_integrations/electrasmart.markdown [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+In order to setup this integration, you must have access to the phone number used to register in the Electra Smart mobile app.
```
Comment (frenck) [inline source/_integrations/electrasmart.markdown:20]: - Suggestion: `In order to` can be written simpler to as `To`
- Typo: `setup` -> `set up`

**Review by frenck — COMMENTED** [review]
Please address these review comments in a new PR.

Thanks! 👍 

../Frenck

File: source/_integrations/electrasmart.markdown (hunk 3 above)
Comment (thecode) [reply source/_integrations/electrasmart.markdown:6]: I had this in mind that we need to fix it before release and forgot. @jafar-atili is on a short vacation, I'll address the comments in a new PR

**Pull request merged**